src/front/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "front",
   "version": "0.1.0",
   "private": true,
+  "homepage": ".",
```

Comment:
because I need to build the front with a relative link from index.html to css and script
otherwise, it uses a link from the root "/" and doesn't find the js and css